docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Sample-based quantum diagonalization
+description: Overview of the Sample-based quantum diagonalization (SQD) workflow
+---
+
+# Sample-based quantum diagonalization (SQD)
+
+Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as the Hamiltonian of a quantum system, using quantum and distributed classical computing together.
```

Comment:
Is there somewhere in this intro that we could add something about why a user might want to use this? the intro does a good job of explaining _what_ the technique is, but not very much about _why_ it might be beneficial for a user